addons/mail/static/src/core/common/store_service.js

```diff
@@ -83,6 +83,8 @@ export class Store extends BaseStore {
     internalUserGroupId;
     /** @type {number} */
     mt_comment_id;
+    /** @type {number} */
+    mt_note_id;
```

Comment:
These 2 fields should now be stored as record instead (and compared as records in getters).